test/test_poolmanager.py

```diff
@@ -392,3 +399,30 @@ def test_pool_manager_no_url_absolute_form(self):
         p = PoolManager()
         assert p._proxy_requires_url_absolute_form("http://example.com") is False
         assert p._proxy_requires_url_absolute_form("https://example.com") is False
+
+    def test_poolmanager_normalizer_default_blocksize(self):
+        """Assert normalizer defaults to blocksize DEFAULT_BLOCKSIZE(16384)"""
+        p = PoolManager()
+
+        pool_blocksize_default = p.connection_from_url(
+            "http://example.com", {"blocksize": DEFAULT_BLOCKSIZE}
+        )
+        assert pool_blocksize_default.conn_kw["blocksize"] == DEFAULT_BLOCKSIZE
+
+        pool_blocksize_not_set = p.connection_from_url("http://example.com")
+        assert pool_blocksize_not_set == pool_blocksize_default
```

Comment:
I am thinking, can we use `pytest.parametrize` to parametrize the `blocksize` and have a single method that can run all the 3 tests ?
Probably do these asserting on all the three cases
```
        assert pool_blocksize_8192.conn_kw["blocksize"] == blocksize
        assert pool_blocksize_8192._get_conn().blocksize == blocksize
```
Reference: https://docs.pytest.org/en/latest/how-to/parametrize.html

Reply:
Makes sense, will do the refactor.